code/game/objects/items/weapons/material/material_armor.dm

```diff
@@ -288,7 +288,9 @@ Protectiveness | Armor %
 
 /obj/item/weapon/material/armor_plating/insert
 	unbreakable = FALSE
-
+	name = "plate insert"
+	desc = "used to craft armor plates for a plate carrier. Trim with a weilder for light armor or add a second for heavy armor"
```

Comment:
```suggestion
	desc = "used to craft armor plates for a plate carrier. Trim with a welder for light armor or add a second for heavy armor"
```
Spelling fixy

Reply:
For the record, there's an option with these suggestions that allow you to merge the offered changes with a click of a button. For now, this has been done otherwise.